templates/lobby.html

```diff
@@ -485,6 +491,7 @@
             playWav('/resources/plop.wav');
         } else if (parsed.type === "update-wordhint") {
             applyWordHints(parsed.data);
+            turnDialog.style.visibility = "hidden";
```

Comment:
This is a weird place to put this, as it's based on an assumption. While this is probably gonna work, it's weird either way. You should at least put a comment in this line, explaining why this is here and why it's okay.

I was actually planning to make some new events and some better turn-flow, this is an ongoing process on my side though.

Reply:
Yes I think too, using existing events, that was the one that seemed to be to most appropriated to me for that.